src/cli/node/with_node.rs

```diff
@@ -220,6 +234,25 @@ pub enum Command {
         #[structopt(long = "limit", allow_hyphen_values = true, default_value = "-50")]
         limit: i64,
     },
+    #[structopt(
+        name = "minerList",
+        alias = "getMiner",
+        about = "List block hashes with their miners and the total number of mined block by each address"
+    )]
+    GetMiners {
+        /// Socket address of the Witnet node to query
+        #[structopt(short = "n", long = "node")]
+        node: Option<SocketAddr>,
+        /// init
+        #[structopt(long = "init", alias = "i")]
+        init: i64,
```

Comment:
I think it is customary to call this `start` as the opposite of `end`.
```suggestion
        #[structopt(long = "start", alias = "s")]
        start: i64,
```